### PR TITLE
style-guide.md: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

- Restrict alternative count to 5, not to permit too long commands like [this](https://github.com/tldr-pages/tldr/pull/9708) one.
- We are not man pages to always be comprehensive, we are short and readable pages.

### CODE REVIEW

File: contributing-guides/style-guide.md [hunk 1]
```diff
@@ -152,6 +152,7 @@ Keep the following guidelines in mind when choosing placeholders:
   and use the `{{/dev/sdXY}}` placeholder for *block devices* instead of `/dev/sda1`.
 - If a command can take a variable number of arguments, use an ellipsis: `{{arg1 arg2 ...}}`.
   If one of the multiple options is possible, write it as `{{either|or}}`.
+  If there are more than 5 alternatives use `|...` after the 5-th alternative.
```
Comment (blueskyson) [inline contributing-guides/style-guide.md:155]:
```suggestion
  If there are many alternatives, use `|...` for ellipsis.
```

Comment (kbdharun) [reply contributing-guides/style-guide.md:155]: How about adding an article `the` before ellipsis. E.g 
> If there are many alternatives, use `|...` for the ellipsis

Comment (blueskyson) [reply contributing-guides/style-guide.md:155]: Updated

Comment (EmilySeville7cfg) [reply contributing-guides/style-guide.md:155]: I've changed it to:

> If there are more than 5 alternatives, use `|...` for the ellipsis.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**